client/my-sites/media-library/filter-bar.jsx

```diff
@@ -151,7 +151,11 @@ export class MediaLibraryFilterBar extends Component {
 	render() {
 		return (
 			<div className="media-library__filter-bar">
-				<SectionNav selectedText={ this.getFilterLabel( this.props.filter ) } hasSearch={ true }>
+				<SectionNav
+					selectedText={ this.getFilterLabel( this.props.filter ) }
+					hasSearch={ true }
+					allowDropdown={ ! this.props.source }
```

Comment:
Could we maybe add an explanatory comment here, just to make it more clear when the dropdown is hidden.